src/Sortable.js

```diff
@@ -1692,7 +1692,7 @@ Sortable.prototype = /** @lends Sortable.prototype */ {
 			if (Sortable.eventCanceled) return;
 
 			// show clone at dragEl or original position
-			if (rootEl.contains(dragEl) && !this.options.group.revertClone) {
+			if (dragEl.parentNode == rootEl && !this.options.group.revertClone) {
```

Comment:
```suggestion
			if (  (rootEl.contains(dragEl) || dragEl.parentNode == rootEl ) && !this.options.group.revertClone) {
```

I have a feeling this might be safer. I'm unsure what remove the old check might do.

Reply:
in my opinion the change above is equivalent to the original so doesn't solve the issue.
I try to better explain my proposal:

original) "_rootEl.contains(dragEl)_" means dragEl is a descendent of Root so it can be child, child of child etc...the condition is too broad and results in an error when dragEl is not just a child.

modified) "_dragEl.parentNode == rootEl_" means dragEl is child of root. That's exactly the condition you have to check before "_rootEl.insertBefore(cloneEl, dragEl)_", if the condition is not met the code will simply move on to one of the following methods to find the right insertion point.

Reply:
Make sense to me